tasks/nonograms/README.md

```diff
@@ -73,18 +73,18 @@ Recommended usage of [eslint (eslint-config-airbnb-base)](https://eslint.org/),
 
 - [ ] layout, design, responsive UI: `+20`
 - [ ] at the beginning state of the game, the frame has size 5x5. The sequence of numbers is logically arranged and help the player solve the nonogram: `+20`
-- [ ] every 5 cells in a rows and columns (both player's field and clues) should be divided by divider `+5`
+- [ ] cells and clues are divided by divides as described in Basic block: `+5`
```

Comment:
`divides` should be replaced with `divider` if I understand correctly

Reply:
dividers, you're right